test/e2e/storage/testsuites/volume_expand.go

```diff
@@ -420,3 +425,10 @@ func WaitForFSResize(pvc *v1.PersistentVolumeClaim, c clientset.Interface) (*v1.
 	}
 	return updatedPVC, nil
 }
+
+func getTestImage() int {
```

Comment:
Is it possible to do this inside e2epod instead of potentially every testsuite?

Reply:
right now no, there is an import cycle issue for e2epod and volume. 
https://github.com/kubernetes/kubernetes/issues/81245